tools/ecosystem/pgbackrest/Makefile

```
@@ -0,0 +1,56 @@
+PYTHON ?= python3
+COMPOSE ?= docker compose
+HARNESS := $(PYTHON) backup_harness.py
+
+.PHONY: build config preflight up stanza check full diff incr info audit verify restore-verify logs down clean test
+
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)